pkg/controller/direct/compute/targettcpproxy/targettcpproxy_controller.go

```diff
@@ -265,15 +265,33 @@ func (a *targetTCPProxyAdapter) Update(ctx context.Context, updateOp *directbase
 	updated := &computepb.TargetTcpProxy{}
```

Comment:
Not a blocker: Maybe declare `op` and `updated` closer to their usage? I think they don't need to be declared explicitly, right?